### ISSUE TITLE
Keyboard Accessibility of ngb-dropdown

### ISSUE THREAD

**Issue by radhikachitkara** [ng-bootstrap/ng-bootstrap#4460]
Hi Team,
 We are facing similar issue, autoclose doesn't work with keyboard navigation.

![image](https://user-images.githubusercontent.com/119498590/209284196-151e136b-7b1c-4ff9-bd38-d33d8df4da86.png)

Below is HTML structure:
`<div class="cart-badge d-md-none text-right flex-grow-1"
        [ngClass]="{ 'nav-cart-mobile': myCart.isOpen() }" #myCart="ngbDropdown" autoClose="true">
        <a class="nav-link text-info" id="dropdownBasic6" (click)="closeOtherDropdowns(myCart)" [attr.aria-expanded]="myCart.isOpen()" role="button" ngbDropdownToggle>
          <i class="aon-icon aon-cart m-r-8" [ngClass]="{ 'cart-icon-mobile': myCart.isOpen() }" aria-hidden="true"></i>
          <div class="icon-badge" >
            <span class="badge badge-danger">5</span>
          </div>
        </a>
        <div ngbDropdownMenu aria-labelledby="dropdownBasic6" class="cart-dropdown">
          <app-masthead-cart [resourcestrings]="resourceStrings" (onCartUpdated)="onCartUpdated($event)"
            (hideCartPopUp)="myCart.close()"></app-masthead-cart>
        </div>
      </div>`

_Originally posted by @radhikachitkara in https://github.com/ng-bootstrap/ng-bootstrap/issues/3279#issuecomment-1363664567_

**Comment by maxokorokov** [comment]
Please take a look at the [Getting Help](https://github.com/ng-bootstrap/ng-bootstrap/blob/master/README.md#getting-help) section in documentation and use the appropriate support channel.

If you believe you've found a bug in this library then please:
- fill in the issue template correctly
- provide a **minimal** reproduction scenario in a StackBlitz

**Comment by maxokorokov** [comment]
Closing as no response, please reopen if necessary.
